whitepapers/0001_generic_message_passing.md

```diff
@@ -189,12 +189,45 @@ level meanings and enforcing them.
 
 - `200` - publish immediately
 - `201` - `safe`, if available, otherwise falls back to `finalized`
+- `202` - `finalized`
+- `203` - custom handling
 - anything else is treated as `finalized`
```

Comment:
Does this statement still make sense?  Otherwise, why have `202`?

Reply:
I think so, it’s just stating that 202 is the actual enum for finalized but finalized is also the fallback value.